gapis/api/templates/mutate.go.tmpl

```diff
@@ -266,29 +266,6 @@
 {{end}}
 
 
-{{/*
```

Comment:
Have we actually been not using this anywhere?

Reply:
Nope. There's a similar function in `api.go.tmpl` that I think you added after this was written.